docker/1.15.2/py3/Dockerfile.cpu

```diff
@@ -99,7 +100,7 @@ RUN pip install --no-cache-dir -U \
     keras_preprocessing==1.1.0 \
     keras==2.3.1 \
     requests==2.22.0 \
-    smdebug==0.5.0.post0 \
+    ${SMDEBUG_URL} \
```

Comment:
Is there an estimate on when this change will be pushed to PyPI?

Reply:
we are targeting march 6